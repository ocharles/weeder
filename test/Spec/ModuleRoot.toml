roots = []

root-modules = [ '^Spec\.ModuleRoot\.M$', '^Spec\.ModuleRoot\.InstanceNotRoot$' ]

type-class-roots = false
