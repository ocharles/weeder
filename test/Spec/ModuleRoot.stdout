main: test/Spec/ModuleRoot/InstanceNotRoot.hs:9:1: (Instance) :: C T
main: test/Spec/ModuleRoot/M.hs:11:1: weed
